jwst/outlier_detection/utils.py

```diff
@@ -64,13 +80,29 @@ def median_without_resampling(input_models,
         if True, save the drizzled models and median model to fits.
 
     make_output_path : function
-        The functools.partial instance to pass to save_median. Must be 
+        The functools.partial instance to pass to save_median. Must be
         specified if save_intermediate_results is True. Default None.
 
     buffer_size : int
-        The size of chunk in bytes that will be read into memory when computing the median.
```

Comment:
Random extraneous comment, but can we make the line length limit 110 characters instead of 80 when we implement the ruff check?  stcal has 110 characters, and personally I find 80 to sometimes detract from readability